Xwt/Xwt/CursorType.cs

```diff
@@ -53,13 +53,17 @@ internal CursorType (string id)
 		public static readonly CursorType ResizeUp = new CursorType ("ResizeUp");
 		public static readonly CursorType ResizeDown = new CursorType ("ResizeDown");
 		public static readonly CursorType ResizeUpDown = new CursorType ("ResizeUpDown");
+		public static readonly CursorType ResizeNESW = new CursorType("ResizeNESW");
+		public static readonly CursorType ResizeNWSE = new CursorType("ResizeNWSE");
 		public static readonly CursorType Hand = new CursorType ("Hand");
+		public static readonly CursorType OpenHand = new CursorType("OpenHand");
 		public static readonly CursorType Move = new CursorType ("Move");
 		public static readonly CursorType Wait = new CursorType ("Watch");
 		public static readonly CursorType Help = new CursorType ("Help");
 		public static readonly CursorType Invisible = new CursorType ("Invisible");
+		public static readonly CursorType Drag = new CursorType("Drag");
```

Comment:
Could you also add Gtk (on Linux it's always mapped to `Gdk.CursorType.Hand2`)/Wpf (?) mappings for `CursorType.Drag`?